vud_widget/src/Plugin/VoteUpDownWidgetInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\vud_widget\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Vote Up/Down Widget plugins.
+ */
+interface VoteUpDownWidgetInterface extends PluginInspectionInterface
```

Comment:
Please see all the calls to `ctools_plugin_get_function()` in D7, that may help to find the missing methods here.